src/kaleidoscope/event_handler_result.h

```diff
@@ -21,6 +21,7 @@ namespace kaleidoscope {
 enum class EventHandlerResult {
   OK,
   EVENT_CONSUMED,
+  ABORT,
```

Comment:
Can you add the doc from that commit message as comments on the ENUM? I want to make sure it's captured somewhere less ephemeral than git history and the changelog.

Reply:
Excellent point. I will add those comments when I get a chance. Thank you.

Reply:
I added some comments describing the return values, and their interpretation.